### PR TITLE
F.Exports -> subclass of list

### PR DESCRIPTION
**Pull request by KulaginVladimir**
## Proposed changes

This PR is the second step of #494 and modifies F.Exports to be a subclass of list. 
Do we need any tests? Maybe, similar to #694? 

## Types of changes

What types of changes does your code introduce to FESTIM?

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Code refactoring
- [ ] Documentation Update (if none of the other choices apply)
- [ ] New tests

## Checklist

- [x] Black formatted
- [x] Unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

### CODE REVIEW

File: festim/h_transport_problem.py [hunk 1]
```diff
@@ -320,7 +320,7 @@ def need_projecting_solute(self, exports):
             bool: True if the solute needs to be projected, False else.
         """
         need_solute = False  # initialises to false
-        for export in exports.exports:
+        for export in exports:
             if isinstance(export, festim.DerivedQuantities):
                 for quantity in export.derived_quantities:
                     if isinstance(quantity, festim.SurfaceFlux):
```
Comment (KulaginVladimir) [inline festim/h_transport_problem.py:326]: @RemDelaporteMathurin here appears a question on this method which is not tested and not used (?)

Comment (RemDelaporteMathurin) [reply festim/h_transport_problem.py:326]: you mean it's not even used in the main branch?

Comment (RemDelaporteMathurin) [reply festim/h_transport_problem.py:326]: Oh I see on line 311 that we have a TODO to remove this "unused" method, so maybe we can take this opportunity to remove it!

**Review by RemDelaporteMathurin — APPROVED** [review]
Thanks @KulaginVladimir ! 🎉

**Pull request merged**